packages/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.ts

```diff
@@ -238,8 +239,14 @@ export default class MeetingInfoV2 {
     } = null,
     installedOrgID = null,
     locusId = null,
-    extraParams = {}
+    extraParams: {meetingId?: string; [key: string]: string} = {}
   ) {
+    let meetingId: string;
+    if (extraParams.meetingId) {
+      meetingId = extraParams.meetingId;
+      delete extraParams.meetingId;
```

Comment:
I think the purpose of extraParams was that these are all params passed in the request to wbxappapi, although not sure if this is documented, so I'm not sure if it's a great idea to be putting the meetingId into them and then stripping it so that it doesn't get sent out in the request. Maybe it should be passed as a separate parameter. You can check with Martin who introduced the extraParams recently.

Reply:
The intention of `extraParams` was to provide extra parameters to the API call, so if the user adds a parameter to this object, they know it will end up being sent to the service. It isn't well documented as you say, but the use of the phrase "params" should be enough to know that they aren't just "options", which is what you effectively change them into. If we make this change, this also opens the door to similar additions in future, which will further change the meaning of params.

Personally, I'd prefer for this to be an extra argument to the function, instead of overloading the `extraParams` argument.

Reply:
addressed